db/seeds/02_users.seeds.rb

```diff
@@ -1,10 +1,15 @@
 # frozen_string_literal: true
 
-puts 'Create users'
+puts '~> Create users'
 
-10.times do
+10.times do |i|
   user = FactoryBot.create(:user, password: '12345678', password_confirmation: '12345678')
-  user.add_role(:user)
+  user.add_role(:user) unless i.zero?
   user.skip_confirmation!
+  user = User.find_by(email: 'admin@trung.com')
+  user.roles.delete(Role.find_by(name: 'user'))
```

Comment:
???
we have method remove_role(:user) (Im not sure, check rolify gem)
But why we have this line?